ProjectSettings/TagManager.asset

```diff
@@ -25,6 +25,10 @@ TagManager:
   - Cave_Worm
   - Spawn_Chest
   - Dev_Teleport
+  - Spawn_Enemy_Snake
+  - Spawn_Ore_Sliding_Barrier
+  - Spawn_Chest_Challenge
+  - Spawn_Chest_Gamble
```

Comment:
So many tags it's making me dizzy. Not your fault it's this way, but has me thinking there could be a better way.

Chests, for example, you entered with all the same spawn point parameters, the only difference is the prefab.
Same with the different ore barriers.

If the spawn point could specify the prefab, wouldn't need so many tags.

Reply:
I guess you'd still need some kind of id, since most of the spawning sorts it by tag